lib/wibox/widget/systray.lua

```diff
@@ -66,6 +66,7 @@ function systray:draw(context, cr, width, height)
     local cols = math.ceil(num_entries / rows)
     local bg = beautiful.bg_systray or beautiful.bg_normal or "#000000"
     local spacing = beautiful.systray_icon_spacing or 0
+    local y_offset = ((height - base_size) / 2) - 1
```

Comment:
I think `base_size` is `nil` if the user hasn't manually changed it before.

Reply:
The CI confirms that 

```
2023-10-02T18:16:23.2795466Z 56 tests finished.
2023-10-02T18:16:23.2796918Z There were 1 errors:
2023-10-02T18:16:23.2806984Z  - /home/runner/work/awesome/awesome/tests/test-systray.lua: 2023-10-02 18:16:16 E: awesome: Error during a protected call: lib/wibox/widget/systray.lua:69: attempt to perform arithmetic on a nil value (upvalue 'base_size')
```

Reply:
@Aire-One  i'm not sure if it has something to do with DPI support across GUI toolkits, or otherwise related to systray implementation - but the problem indeed real, some apps getting smaller icons than others

Reply:
oops, wrong thread, i thought i'm replying to https://github.com/awesomeWM/awesome/pull/3860#pullrequestreview-1653591164